content/post/dataproc_lzo.md

```diff
@@ -0,0 +1,51 @@
+---
+author: vadim
+date: 2016-10-21
+title: "Adding LZO support to Dataproc"
+draft: false
+image: /post/dataproc_lzo/intro.png
+description: "How to leverage splittable LZO compression in Dataproc"
+---
+[LZO](https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Oberhumer) is a nice
+general-purpose data compression algorithm which provides blazing fast decompression.
+It is especially good because LZO archives can be indexed and become
+[splittable](http://stackoverflow.com/questions/34208051/when-are-files-splittable).
+source{d} uses LZO to store the dataset of names inside GitHub repositories,
```

Comment:
I would explain what kind of names, because it sounds like names of the repositories.